Governance.md

```diff
@@ -0,0 +1,14 @@
+Governance
+==========
+On 8th December 2015, 17 members of the active community voted to elect the first dRonin Executive Committee.
+
+Those voted in were: hshred, icee, tracernz.
```

Comment:
Let's use github user names or real names. Also, should we call it Executive Board? Sounds more professional (committee also sounds like it won't get anything done ;)